TestingTravis/TestClass.swift

```diff
@@ -17,6 +17,7 @@ class TestClass: NSObject {
         print(forceUnwrapMe!)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/2e2e460f3810119f6af8655b5fb0f92a2525248b/TestingTravis/TestClass.swift#L17"></span>Force unwrapping should be avoided.<br />
<code>force_unwrapping</code> <code>TestClass.swift:17</code></td>
    </tr>
  </tbody>
</table>